sig-windows/README.md

```diff
@@ -31,6 +31,10 @@ The Chairs of the SIG run operations and processes governing the SIG.
 ## Subprojects
 
 The following subprojects are owned by sig-windows:
+- **windows-gmsa**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-sigs/windows-gmsa/master/OWNERS
+
```

Comment:
this file is generated and it won't pass CI if modified incorrectly:

> Verifying verify-generated-docs.sh
I0214 22:46:08.250] /go/src/k8s.io/community/sig-windows/README.md does not match /tmp/tmp.QLAz4oQ4uB/src/testgendocs/sig-windows/README.md

> I0214 22:46:08.266] 1 mismatches were detected.
I0214 22:46:08.266] Do not manually edit sig-list.md or README.md files inside the sig folders.
I0214 22:46:08.267] Instead make your changes to sigs.yaml, then run `make`, and then
I0214 22:46:08.267] commit your changes to sigs.yaml and any generated docs.

`./hack/verify-generated-docs.sh` can be used locally to check.

Reply:
i messed up and forgot to generate it. i just generated it now @neolit123